src/loadPackageDefinition.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path');
+const grpc = require('grpc');
+const protoLoader = require('@grpc/proto-loader');
+const snakeCase = require('lodash.snakecase');
+
+function loadPackageDefinition(serviceName) {
```

Comment:
I think we need to move all shared with dapi-grpc code into a separate package `grpc-utils` or something like that.

Reply:
@shumkov I think it is better to postpone it and implement as a separate story, since it turned out to be not so easy.